pkg/build/pipelines/go/build.yaml

```diff
@@ -0,0 +1,55 @@
+name: Run a build using the go compiler
+
+needs:
+  packages:
+    - go
+    - busybox
+    - ca-certificates-bundle
+
+inputs:
+  package:
+    description: |
+      List of packages to compile
```

Comment:
A few thoughts about user clarity...

1. If it's a list of packages, maybe the input name should be "packages"?
2. Should we clarify that this list should be _space-separated_ (I'm assuming this based on how we're passing this input to `go build` below.)
3. Should we mention in the description that files are also acceptable (I noticed the example has `package: main.go`, for instance.)

Reply:
Yeah sounds good to me, one of the principles I'm trying to follow is to keep the language and terms the underlying compiler uses (go in this case). So I'm just relaying words :)

Reply:
The updates look great! 🎉